client/app/src/accounts/AccountController.js

```diff
@@ -129,6 +129,20 @@
       else return null;
     };
 
+    self.clearData = function() {
+      var confirm = $mdDialog.confirm()
+            .title('Are you sure?')
+            .textContent('All your data, including created accounts, networks and contacts will be lost and reset to default.')
```

Comment:
`lost` is too strong here.
I would say `All your data, including created accounts, networks and contacts will be removed from the app and reset to default.` instead